static/js/around-map.js

```diff
@@ -107,3 +109,19 @@ function urlFor (id) {
 function findPoi (id) {
   return pois.find(poi => poi._id == id)
 }
+
+function highligth(poi_id) {
+  const poiID = poi_id
+  const cardActive = document.getElementsByClassName('currentCard')
+  if (cardActive[0]) {
+    cardActive[0].classList.toggle('currentCard')
+  }
+  const card = document.getElementById('card_'+poiID)
+  if (card == null) {
+    const cardNull = setInterval( _ => {
+      highligth()
```

Comment:
I'm pretty sure this could be solved with an asynchronous event.
We should talk about it to make sure what you're attempting to do.